### DISCUSSION TITLE
Set cost per location and time of day

### DISCUSSION THREAD

**Discussion by tontonchow** [teslamate-org/teslamate#1509] [Ideas]
Hello,
I am looking for the possibility to define the cost of KWH according to the time of the day and the place where I am (the idea being to see the price of the supercharge and charge at home in automatic). In France we have different costs according to daytime/nighttime and we also have many charging point with different cost per KWH

**Comment by Jessblog** [comment]
Hello,
+1, french mate here, looking forward to have this time slots differentiation.

**Comment by mrmontecristo82** [comment]
Hi,
+1, Portuguese fellow. Not including the Market Rate Indexed energy tariff plans (which could be quite complicated to set up, since they can vary per hour / per day, continuously), we usually have tariff plans with 3 different slots per day (off peak, peak, "high peak"), and the times for these slots can change each day.

**Comment by Alabresque** [comment]
Same here in Spain, different prices in SuCs and at home we have 3 periods in weekdays. Would be great to have that option.

**Comment by winnerwq** [comment]
+1, Chinese follow.  We also have different price at different time of the day.

**Comment by Amourgo** [comment]
+1, Chinese follow. We also have different price at different time of the day.

**Comment by damccull** [comment]
This might be better opened as a Feature Request in issues, rather than here in discussions.

**Comment by gzl-tg** [comment]
+1, Chinese follow.

**Comment by acravid** [comment]
+1 Portuguese fellow.
